package.json

```diff
@@ -34,7 +34,7 @@
     "prettier": "prettier './src/**/*.{js,jsx,ts,tsx}'",
     "start": "rescripts start",
     "start:docker": "docker-compose build && docker-compose up",
-    "test": "REACT_APP_ENV=test rescripts test --env=jsdom",
+    "test": "cross-env REACT_APP_ENV=test rescripts test --env=jsdom",
```

Comment:
This was added for better testing capability on Windows. I have checked with @usame-algan for Mac and @dasanra for Linux capability.